core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/PropertiesUtils.java

```diff
@@ -470,6 +479,29 @@ public static void updateURIParams(AbstractCamelModelElement selectedEP, Paramet
     	}
     }
     
+    /**
+     * Replaces either the whole part of a given text or nothing. The parts are determined by the specified delimiter.
+     * 
+     * @param text text
+     * @param target string to be replaced
+     * @param replacement replacement string
+     * @param delimiter delimiter
+     * @return text with replaced parts  
+     */
+    public static String replaceParts(String text, String target, String replacement, String delimiter) {
+    	String[] parts = text.split(delimiter);
+		for (int i = 0; i < parts.length; i++) {
```

Comment:
seems ok now :-)
it is effectively better to avoid changes everywhere only for formatting reasons.
To apply formatting only on few lines you can select the lines and then use Ctrl+Shift+F